Client/Frontend/Browser/Playlist/Utilities/PlaylistMediaStreamer.swift

```diff
@@ -99,7 +99,7 @@ class PlaylistMediaStreamer {
           // If we don't do this, youtube shows ads 100% of the time.
           // It's some weird race-condition in WKWebView where the content blockers may not load until
           // The WebView is visible!
-          self.playerView?.window?.insertSubview($0, at: 0)
+          self.playerView?.window?.insertSubview($0, at: 0) ?? UIApplication.shared.keyWindow?.insertSubview($0, at: 0)
```

Comment:
Same comment as above

Reply:
For this case, you can see an attempt to add it to the current window. If it cannot, it falls back to keyWindow. This is because CarPlay has no windows and there's no other way to force the web-view to load its page. Youtube will legit not work as it checks if the page is visible before it does anything.

Reply:
Fixed